app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomeDestination.kt

```diff
@@ -355,9 +393,65 @@ private fun HomeScreenSuccess(
   }
 }
 
+@Composable
+private fun ImportantMessages(
+  list: List<HomeData.VeryImportantMessage>,
+  openUrl: (String) -> Unit,
+  hideImportantMessage: (id: String) -> Unit,
+  modifier: Modifier = Modifier,
+) {
+  AnimatedContent(targetState = list) { animatedList ->
+    var consumedWindowInsets by remember { mutableStateOf(WindowInsets(0.dp)) }
+    val contentPadding = PaddingValues(horizontal = 16.dp) + WindowInsets.safeDrawing
+      .exclude(consumedWindowInsets)
+      .only(WindowInsetsSides.Horizontal)
+      .asPaddingValues()
+    val updatedModifier = Modifier.onConsumedWindowInsetsChanged { consumedWindowInsets = it }
```

Comment:
This onConsumedWindowInsetsChanged modifier (+ the consumedWindowInsets state) can also just be applied on the AnimatedContent itself, so that it happens in one spot at the top and there's no need to pass it down to the two possible cases